act-rules-format.bs

```diff
@@ -52,66 +48,91 @@ The separation between atomic rules and composite rules creates a division of re
     <li>Header indicated by using the `headers` attribute</li>
     <li>Header indicated by using ARIA labels</li>
   </ul>
-  <p>If any one of these rules passes, the test result of the composite rule for the table cell is "passed".</p>
+  <p>If any one of these rules passes, the outcome of the composite rule for the table cell is "passed".</p>
 </div>
 
+The separation between atomic rules and composite rules creates a division of responsibility. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules test if a combination of outcomes from atomic rules is sufficient for an accessibility requirement to be conformant or not.
+
+Not all atomic rules have to be part of a composite rule. Only if the outcome of multiple atomic rules has to be combined in order to determine (non-)conformance of the [Accessibility Requirements](#accessibility-requirements) can composite rules be used. An atomic rule MAY be part of multiple composite rules.
```

Comment:
"Only if the outcome of multiple atomic rules has to be combined in order to determine (non-)conformance of the [Accessibility Requirements](#accessibility-requirements) can composite rules be used" -> "Composite rules are used when the outcome of multiple atomic rules needs to be combined in order to determine (non-)conformance [of a test subject] to the [Accessibility Requirements](#accessibility-requirements)"